Sources/XcbeautifyLib/Renderers/TerminalRenderer.swift

```diff
@@ -140,11 +140,20 @@ struct TerminalRenderer: OutputRendering {
         return colored ? "\(Symbol.error) \(reason.f.Red)" : "\(Symbol.asciiError) \(reason)"
     }
 
-    // TODO: Print file path
     func formatLinkerDuplicateSymbolsError(group: LinkerDuplicateSymbolsCaptureGroup) -> String {
         let reason = group.reason
         return colored ? "\(Symbol.error) \(reason.f.Red)" : "\(Symbol.asciiError) \(reason)"
     }
+    
+    func formatLinkerDuplicateSymbolsLocation(group: LinkerDuplicateSymbolsLocationCaptureGroup) -> String? {
+        let filePath = group.filePath
+        return Format.indent + (colored ? "\(filePath.f.Red)" : "\(filePath)")
```

Comment:
I think we want to prefix `Symbol.error` and `Symbol.asciiError` to these messages, right?

Reply:
I think error messages with symbol prefixes is a little bit distractive. But prefix it with a symbol can be more integrity.

I'll fix it.